AppCenterAnalytics/AppCenterAnalytics/MSAnalytics.m

```diff
@@ -153,7 +158,12 @@ - (void)updateConfigurationWithAppSecret:(NSString *)appSecret transmissionTarge
 
   // Create the default target if not already created in start.
   if (token && !self.defaultTransmissionTarget) {
-    self.defaultTransmissionTarget = [self transmissionTargetForToken:token];
+
+    /*
+     * Don't use [self transmissionTargetForToken] because that will add the default transmission target to the cache, but it should be
+     * separate.
+     */
+    self.defaultTransmissionTarget = [self createTransmissionTargetForToken:token];
```

Comment:
This isn't covered by unit test. Please add one.

Reply:
This wasn't covered before the change and I'm not actually sure that it can be hit.  Will require further investigation but is out of scope for now.